<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="class" stdset="0">
       <string>deep</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="heightEdit">
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0.5, 2, 0.5, 0.2, 1, 1, 0.2, 0, 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="shaderCombo"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="cumulativeLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximum">
       <number>8192</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="maximum">
       <number>8192</number>
      </property>
      <property name="value">
       <number>8192</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>shaderCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>setShader(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>setColormap(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>-30</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setPositionLeft(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setPositionRight(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setShader(QString)</slot>
  <slot>setColormap(QString)</slot>
  <slot>setPositionLeft(int)</slot>
  <slot>setPositionRight(int)</slot>
 </slots>
</ui>
